.github/workflows/advanced.yml

```diff
@@ -14,7 +14,7 @@ on:
       gcc_branch:
         description: 'GCC branch to build'
         required: false
-        default: 'woarm64'
+        default: 'fix-enable-shared'
```

Comment:
Shall be reverted once https://github.com/Windows-on-ARM-Experiments/gcc-woarm64/pull/14 will be merged.